packages/main/src/plugin/proxy-resolver.ts

```diff
@@ -140,8 +142,7 @@ export function createHttpPatch(originals: typeof http | typeof https, proxy: Pr
   }
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function createHttpPatchedModules(proxy: Proxy, certificates: Certificates): any {
+export function createHttpPatchedModules(proxy: Proxy, certificates: Certificates): Record<string, object> {
```

Comment:
why 'object' ? there is `http` and `https`

Reply:
`originals: typeof http | typeof https` maybe? otherwise, rest of test LGTM.